kube/src/api/typed.rs

```diff
@@ -195,19 +195,10 @@ where
 
     /// Patch a resource a subset of its properties
     ///
-    /// Note that some of the [`PatchStrategy`](crate::api::PatchStrategy) variants require different serialization.
-    ///
-    /// The original strategies such as `PatchStrategy::Merge`, `PatchStrategy::JSON`
-    /// and `PatchStrategy::Merge` should be serialized using `serde_json::to_vec`.
-    ///
-    /// See [kubernetes json patch types](https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/#use-a-json-merge-patch-to-update-a-deployment)
-    /// for more information about their distinction.
-    ///
-    ///
-    /// When using `PatchStrategy::Apply`, you **must** serialize your
-    /// data using `serde_yaml`:
```

Comment:
Removed repetitive `PatchStrategy` and moved the link to k8s docs after the code block.

<img src="https://user-images.githubusercontent.com/639336/103121001-0965b600-462f-11eb-8e40-f03e50b359e2.png" width=600>

<img src="https://user-images.githubusercontent.com/639336/103121015-1aaec280-462f-11eb-98b0-f02b0980e90d.png" width=600>

Reply:
the new [`Merge`] and [`JSON`] links (below) actually direct into `PatchStrategy`?!

Reply:
Yup

![image](https://user-images.githubusercontent.com/639336/103128707-42138880-464b-11eb-85eb-af6bcaa5577b.png)

Reply:
ah, missed those